docs/DeveloperGuide.md

```diff
@@ -656,29 +758,21 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
 
 ### Deleting a person
 
-1. Deleting a person while all persons are being shown
+1. Deleting a person while all fosterers are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+   1. Prerequisites: List all persons using the `list` or `find` command. At least 6 fosterers in the list.
```

Comment:
why need at least 6 fosterers in the list to delete multiple...?

Reply:
sorry it was because of my test case, ill change it to 3 instead